resources/translations/de_DE/read.me

```diff
@@ -0,0 +1,9 @@
+complete translated folders:
```

Comment:
It's fine to keep track like this, but best not commit those files. You can add them to your .gitignore maybe?

Reply:
yeap added to gitignore